geo-data-server/src/test/java/com/powsybl/geodata/server/GeoDataControllerTest.java

```diff
@@ -72,9 +70,25 @@
     @MockBean
     private LineRepository lineRepository;
 
+    @Autowired
+    private CqlSessionCassandraConnection cqlSessionCassandraConnection;
+
     @Before
-    public void setUp() {
+    public void setup() throws IOException {
```

Comment:
It is duplicated in all unit tests. You have 2 way to fix this:
  - create an abstract class to have a common setup of cassandra.
  - you can also create a junit rule